README.md

```diff
@@ -51,47 +51,87 @@ This project is licensed under the terms of the
 [Storybook](https://storybook.js.org/)  
 [Jest](https://jestjs.io/)
 
-### Installation
+## Installation
 
 Link local packages together and install remaining package dependencies:
 
-```
+```bash
 yarn bootstrap
 ```
 
 Installs with [Lerna bootstrap](https://github.com/lerna/lerna/tree/main/commands/bootstrap#usage)
 
-### Run Storybook
+## Run Storybook
+
+```bash
+cd storybook
+```
+
+install packages
 
+```bash
+yarn
 ```
+
+start srotybook
```

Comment:
*storybook

Reply:
What is this start command? Isn't it `yarn storybook`?

Reply:
yes indeed, that is me thinking something else while typing

Reply:
it is `start-storybook -p 9000`